sys/kern/syscalls.c

```diff
@@ -531,13 +531,23 @@ static int sys_clock_nanosleep(proc_t *p, clock_nanosleep_args_t *args,
   clockid_t clock_id = args->clock_id;
   int flags = args->flags;
   const timespec_t *u_rqtp = args->rqtp;
-  timespec_t rqtp;
-  int error;
+  timespec_t *u_rmtp = args->rmtp;
+  timespec_t rqtp, rmtp;
+  int error, error2;
```

Comment:
I'm not sure if `error2` is the best name for variable.

Reply:
Done.